src/data/faq.json

```diff
@@ -2084,6 +2084,7 @@
     "glossary": {
         "glossaryHeadline": "Glossary",
         "optionalText": "",
+        "linkToGlossary": "Link to the glossary",

```

Comment:
```suggestion
        "linkToGlossary": "Link to the Glossary",
```
Capitalize, since it is a section heading.

Reply:
Changed it, thanks!